README.md

```diff
@@ -1,90 +1,49 @@
-
-1. Words stored in the `sentiment` key of the lexicon.
-1. Punctuation (e.g. exclamation points may increase the sentiment value)
-1. Behavior value
-
-Given these criteria, produce a sentiment value between 0 and 1 for each message.
-
-## Behavior Predictor
-
-This module will determine what behavior is expressed in a message. Use the following criteria.
-
-1. Words stored in the `behavior` key of the lexicon.
-1. Punctuation (e.g. question marks may indicate inquisitiveness, exclamation point may indicate aggressiveness)
-
-Given these criteria, produce a behavior value between 0 and 1 for each message, for each behavior identified.
-
-> **Instructor guidance:** You may want to add more behaviors to the lexicon as you identify them.
-
-## Tokenizer
+### Summary
+---
+This project is a language analyzer that analyzes 2016 presidential nominees' tweets. The program is intended to analyze a twitter message based on categories relating to the presidiential race. The categories used are sentiment, behavior, and domain.
 
-The tokenizer module will produce the following results for each message.
+Sentiment: The sentence gets scanned for the positive, neutral, or negative keywords then produces a value based on the amount of matches found.
 
-1. Alphanumeric Characters
-1. Word count
-1. Word position
-1. Sentence count
-1. Punctuation
+Behavior Predictor: The behavior of the sentence can be predicted to see if it fits into the categories of Dictator, Hippie, Blame, Childish, Mockery, Pandering, Supportive, Promises, and Patriotic.
 
-## Domain Identifier
+Domain: The sentence is evaluated to see if it fits into the topic of Healthcare, War, Climate, and Political Correctness.
 
-The domain identification module will determine the subject matter of the message. Use the following criteria.
+### Example
+---
+![Light Example](/readme_images/light_example.png)
 
-1. Words stored in the `domain` key of the lexicon.
-1. Punctuation
-1. Word count
+### Motivation
+---
 
-Given these criteria, produce a domain value between 0 and 1 for each message, for each domain identified.
+The motivation behind this project is to scan the presidential tweets and help citizens make a more conscious and informed decision for the Fall 2016 election.
 
-> **Instructor guidance:** You may want to add more domains to the lexicon as you identify them.
+### Installation
+---
+If you're on Mac OS X, consider installing with [Homebrew](https://github.com/yyuu/pyenv#homebrew-on-mac-os-x). Run the command ```$ python --version``` from your command line , we recommend Python 3.3.6
 
-##### Example
+If you do not have pyenv use this [link](https://github.com/yyuu/pyenv#homebrew-on-mac-os-x) to install.
 
-> Friendship is like money, easier made than kept.
+This project was built using Python 3.3, click [here](https://www.python.org/download/releases/3.3.0/) to install this version.
 
```

Comment:
right here we could add the following....

 $ git clone https://github.com/nss-day-cohort-13/language-analyzer-lazer-powered-cthulhu.git```

$ cd language-analyzer-lazer-powered-cthulhu```